auth/permisos.ts

```diff
@@ -56,7 +56,8 @@ export default [
                     { key: 'puedeImprimir', title: 'Revisar-Imprimir agenda', comment: 'Habilita botón Auditoría de agendas', type: 'boolean' },
                     { key: 'puedeReasignar', title: 'Reasignar Turno', type: 'boolean' },
                     { key: 'puedeEditarCarpeta', title: 'Editar número de carpeta', type: 'boolean' },
-                    { key: 'puedeBorrar', title: 'Borrar agenda', comment: 'Borra agendas en planificación', type: 'boolean' }
+                    { key: 'puedeBorrar', title: 'Borrar agenda', comment: 'Borra agendas en planificación', type: 'boolean' },
+                    { key: 'READ', title: 'Solo Lectura', comment: 'ver agenda publicada', type: 'boolean' }
```

Comment:
Mover arriba de todo así en el gestor de usuarios se ve primero.

```suggestion
                    { key: 'read', title: 'Ver agendas', comment: 'ver agenda publicada', type: 'boolean' }
```

Reply:
@corregido